packages/uiweb/src/lib/components/space/SpaceWidget/WidgetHeader.tsx

```diff
@@ -0,0 +1,220 @@
+        {isLive &&
+          <Item display={'flex'} alignSelf={'flex-start'} alignItems={'center'} marginLeft={'24px'}>
+            <Image
+              src={SettingsIcon}
+              alt="Settings icon"
+            />
+            <Item marginLeft={'8px'} display={'flex'}>
+              <Image
+                onClick={() => setIsMinimized(!isMinimized)}
+                src={isMinimized ? CaretUpIcon : CaretDownIcon}
+                alt="Maximize/Minimize icon"
+              />
+            </Item>
+            <Item marginLeft={'8px'} display={'flex'} onClick={handleCloseWidget}>
+              <CloseSvg stroke='white' height='15' width='15'/>
+            </Item>
+          </Item>
+          }
+      </Section>
+      {!isLive &&
+        <Item display={'flex'} marginTop={'12px'} alignItems={'center'}>
+          <Image
+            src={CalendarIcon}
+            alt="Calendar Icon"
+          />
+          <Item marginLeft={'4px'} fontSize={'14px'} fontWeight={600}>{formatDate(Date.now())}</Item>
+        </Item>
+      }
+      {isLive &&
+        <Section marginTop='12px'>
+          <Item display={'flex'} alignItems={'center'}>
+            <Image
+              src={LiveIcon}
+              alt="Calendar Icon"
+            />
+            <Text fontSize={'14px'} fontWeight={600} marginLeft={'4px'}>Live</Text>
+          </Item>
+          <Item display={'flex'} alignItems={'center'}>
+            <Item>
+              {/* ToDo: Add participants icons */}
+            </Item>
+            <Text fontSize={'14px'} fontWeight={600} marginLeft={'4px'}>+190 Listeners</Text>
+          </Item>
+        </Section>
+      }
+    </Container>
+  );
+};
+
+//styles
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: white;
+  padding: 16px 24px;
+  background: linear-gradient(87.17deg, #EA4EE4 0%, #D23CDF 0.01%, #8B5CF6 100%), linear-gradient(87.17deg, #EA4E93 0%, #DB2777 0.01%, #9963F7 100%), linear-gradient(87.17deg, #B6A0F5 0%, #F46EF7 50.52%, #FFDED3 100%, #FFCFC5 100%), linear-gradient(0deg, #8B5CF6, #8B5CF6), linear-gradient(87.17deg, #B6A0F5 0%, #F46EF7 57.29%, #FF95D5 100%), #FFFFFF;
+`;
+
+const Image = styled.img`
+  display: flex;
+  max-height: initial;
+  vertical-align: middle;
+  overflow: initial;
+  cursor: pointer;
+  height: ${(props: any): string => props.height || '24px'};
+  width: ${(props: any): string => props.width || '20px'};
+  align-self: center;
+`;
+
+const Section = styled.div<{marginTop?: string}>`
+  display: flex;
+  justify-content: space-between;
+  margin-top: ${(props) => props.marginTop};
+`
+
+const ProfileContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: fit-content;
+  align-items: center;
+  margin-bottom: 12px;
+}`;
+
+const PfpContainer = styled.div`
+  display: flex;
+}`;
+
+const Pfp = styled.img`
+  height: 32px;
+  width: 32px;
+  border-radius: 50%;
+}`;
```

Comment:
I believe this can be reused from @samarendra-push 's banner component

Reply:
good catch, at the time of implementation, banner component wasn't merged, so copied it
will put it in reusables and use it here